deploy/template.yaml

```diff
@@ -1568,6 +1571,60 @@ Resources:
           Period: 60
           Stat: Sum
 
+  FrontRestApiGateway5XXErrorAlarm:
```

Comment:
Does this alarm render the `FrontTargetGroup5xxErrors` redundant?
In prod, we'd need to see two 5 minute periods back to back with more than 50 5xx errors from the target group for that alarm to trigger.
If we had 50 errors in minute 1 and then 50 in minute 6, the new alarm wouldn't trigger. So maybe this is still useful?